test/test-proxied-properties.js

```diff
@@ -140,6 +140,7 @@ describe("browser-polyfill", () => {
   describe("without side effects", () => {
     it("should proxy non-wrapped methods", () => {
       let lazyInitCount = 0;
+      let lazyInitPrivacyCount = 0;
```

Comment:
Could you create a new `describe(...)` block to test the privacy API?

Reply:
Done.